rir/src/api.cpp

```diff
@@ -95,7 +95,7 @@ REXPORT SEXP rir_body(SEXP cls) {
     return f->container();
 }
 
-REXPORT SEXP pir_compile(SEXP what, uint32_t verbose, SEXP dryRun_) {
+REXPORT SEXP pir_compile_(SEXP what, uint32_t verbose, SEXP dryRun_) {
```

Comment:
no REXPORT. This function should not be called from R

Reply:
REXPORT means we want to call this function from R code, using `.Call`. In this case R assumes that it has a signature, which takes only SEXP arguments and returns a SEXP.

A function with uint32_t as argument cannot be REXPORT